df.world-site.xml

```diff
@@ -924,9 +925,21 @@
         <enum-item name='tree_house'/>
         <enum-item name='hillock_house'/>
         <enum-item name='mead_hall'/>
+        20
         <enum-item name='fortress_entrance'/>
         <enum-item name='library'/>
         <enum-item name='tavern'/>
+        <enum-item name='countring_house'/>
```

Comment:
Is this a typo? (I'm not sure, Google isn't returning anything particularly meaningful for this)

Reply:
yeah, it's a type for 'counting house' *sigh*